src/main/drivers/system.c

```diff
@@ -284,3 +284,21 @@ void unusedPinsInit(void)
 {
     IOTraversePins(unusedPinInit);
 }
+
+char *getMCUTypeName(void)
```

Comment:
Not super stoked about introducing a rift between what we report in CLI and in MSP / configurator for this.

How about we introduce a function that resolves the MCU type id, and then a function `getMcuTypeById(mcuId_e id)` to map to name?

Reply:
Not too sure about a significance of `getMcuTypeById`, as there is only one `MCU_TYPE_ID` defined for each build...

Reply:
See #9479 for my proposal.